### PR TITLE
Additional faction info

### PR DESCRIPTION
**Pull request by rzulian**
What about this?

### CODE REVIEW

File: src/components/PlayerInfo.vue [hunk 1]
```diff
@@ -69,7 +69,11 @@ export default class PlayerInfo extends Vue {
   }
 
   get factionDesc() {
-      return `<b>Ability: </b> ${factionDesc[this.player.faction].ability} </br><b>PI: </b> ${factionDesc[this.player.faction].PI} `;
+      return `<b>Cost:</b> ${factionDesc[this.player.faction].cost} </br>
```
Comment (coyotte508) [inline src/components/PlayerInfo.vue]: It seems to be the same as in Commands.vue. Maybe make it a single method and use it at both locations?

File: src/data/factions.ts [hunk 2]
```diff
@@ -1,35 +1,62 @@
...
+export const stdCost = 'm:2c,1o ts:3/6c,2o lab:5c,3o PI:6c,4o AC:6c,6o';
+export const stdIncome = 'm:1o ts:3/4/4/5c lab:1k PI:4o,1t AC1:2k AC2:1q - round:1o,1k';
+export const stdStartIncome = '1o,1k';
 
-export const factionDesc = {
+export const factionDesc =  {
   [Faction.Terrans] : {
+    cost: stdCost,
+    income: stdIncome,
+    startIncome: stdStartIncome,
```
Comment (coyotte508) [inline src/data/factions.ts]: Why repeat for each faction? By the way, might as well use what's in engine with faction boards to  get real data.

Comment (rzulian) [reply src/data/factions.ts]: I've tried to have objects with only the properties need, but was getting an error form vue, stating that not cost was in all objects. I didn't have time to investigate :(

File: src/components/PlayerInfo.vue (hunk 1 above)
Comment (rzulian) [reply src/components/PlayerInfo.vue]: moved to factions

**Comment by coyotte508** [comment]
You need to specify the type:

```
const factionData: { [faction in Faction]: {income?: string, cost?: string, ability: string, PI: string}} = {...};
```

Then you shouldn't have the vue error.

**Comment by coyotte508** [comment]
Well... I wish it was based on the faction data already available in engine... but ok!

**Comment by rzulian** [comment]
@coyotte508 I've tried, but events are not available during faction selection, and we are creating only the choosen factions. Even more, events are changing during the game, and you don't know if they are the setup or the added ones. We should have the event source. Am I wrong?

**Comment by coyotte508** [comment]
In `engine/index.ts`, you can do:

```
import factionBoards from './src/faction-boards';

export {factionBoards}; // Can be combined with other exports
export {factionBoard} from './src/faction-boardss'; // can be combined with export FactionBoard
```

In the viewer, then:

```
import {factionBoards} from '@gaia-project/engine';

function factionDesc(faction: Faction) {
  const board = factionBoard(faction);
  
  const startingIncome = board.income.events.filter(ev => ev.Operator === Operator.Once);
  const roundIncoe = board.income.events.filter(ev => ev.Operator === Operator.Income);

  // ...
}
```

**Pull request merged**